### ISSUE TITLE
112783682 contact reply isn't sending email ? (Julie )

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1108]
Issue reported by **JulieHarold** at 2018-07-29 08:48:02 UTC
Time: 2018-07-29T08:42:11+00:00
Session ID: f9e20d71ef0162d3205f598a99435fa5
Problem Page URL: [/contacts/5b4b52cbf493fdb1c2402510/reply](/contacts/5b4b52cbf493fdb1c2402510/reply)
Previous Page URL: [https://www.freereg.org.uk/contacts](https://www.freereg.org.uk/contacts)
Reported Issue:
Hi, I have corrected a few entries with the new reply function on the website.  The first couple sent me a copy of the email.  The last 3 have not sent me an email, and gave me this error message.  The comments I made however are listed on the view replies section of the contact.

Also a comment - the reply email seems to have lost all formatting on the comment so that it is given as all one line, which is hard to follow.

Also - are we supposed to destroy now ?  I presume not as there is the view replies option.

Julie Harold
Norfolk coordinator

http://www13.freereg.org.uk/uploads/feedback/screenshot/5b5d7f42f493fd75515a56a1/Capture.JPG    

![Screenshot](http://www.freereg.org.uk/uploads/feedback/screenshot/5b5d7f42f493fd75515a56a1/Capture.JPG)

**Comment by Captainkirkdawson** [comment]
Likely being caused by #1514

**Comment by Captainkirkdawson** [comment]
Should have been fixed by the revised contact system
